models/customers-search-list-my-view/src/types.ts

```diff
@@ -0,0 +1,16 @@
+import { TBuilder } from '@commercetools-test-data/core';
+import type { TMcSettingsCustomersSearchListMyView } from '@commercetools-test-data/graphql-types';
+
+export type TCustomersSearchListMyView =
+  TMcSettingsCustomersSearchListMyView & {
+    name: string;
+  };
+
+export type TCustomersSearchListMyViewGraphql = TCustomersSearchListMyView & {
+  __typename: 'CustomersSearchListMyView';
```

Comment:
This property is already declared in the type imported from the `@commercetools-test-data/graphql-types` package ([reference](https://github.com/commercetools/test-data/blob/main/graphql-types/src/generated/settings.ts#L485)).

Reply:
🔍 You got an eagle's eye.  Fixed in https://github.com/commercetools/test-data/pull/654/commits/384d66ea09f895d0ea3607ffc7f1657b4b9f5781